punishments.js

```diff
@@ -24,14 +24,15 @@ const SHAREDIPS_FILE = 'config/sharedips.tsv';
 const RANGELOCK_DURATION = 60 * 60 * 1000; // 1 hour
 const LOCK_DURATION = 48 * 60 * 60 * 1000; // 48 hours
 const BAN_DURATION = 7 * 24 * 60 * 60 * 1000; // 1 week
+const BATTLEBAN_DURATION = 48 * 60 * 60 * 1000; // 48 hours
 
 const ROOMBAN_DURATION = 48 * 60 * 60 * 1000; // 48 hours
 const BLACKLIST_DURATION = 365 * 24 * 60 * 60 * 1000; // 1 year
 
 const USERID_REGEX = /^[a-z0-9]+$/;
 const PUNISH_TRUSTED = false;
 
-const PUNISHMENT_POINT_VALUES = {MUTE: 2, BLACKLIST: 3, ROOMBAN: 4};
+const PUNISHMENT_POINT_VALUES = {MUTE: 2, BLACKLIST: 3, BATTLEBAN: 4, ROOMBAN: 4};
```

Comment:
???

I'm confused, is this a global punishment or a room punishment? I don't think it should count towards the monitor like this if its a global punishment.

Reply:
It’s a global punishment but it’s counting towards a lock if they cause other issues

Reply:
But yeah if we don’t change it so battle bans auto lock, this is undoubtably needed.

Reply:
In its present form, I agree that this should be counted.